react-native/react/router/meta-navigator.js

```diff
@@ -5,18 +5,29 @@
  * todo(mm) explain why we need a meta navigator
  */
 
-import React, { Component } from '../base-react'
-import Render from './meta-navigator-render'
+import React, {Component} from '../base-react'
+import Render from './meta-navigator.render'
 import Immutable from 'immutable'
+import {connect} from '../base-redux'
 
 class MetaNavigator extends Component {
-  constructor () {
-    super()
+  constructor (props) {
+    super(props)
 
+    console.log('NOJ meta-nav will construct')
```

Comment:
- [ ] remove this stuff from this file. just to show when things are made / destroyed. now it happens when we expect!